src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -262,6 +268,25 @@ def get(self, request: Request, project: Project) -> Response:
                     else:
                         scope.set_tag("stacktrace_link.error", "file_not_found")
 
+                result["commitSha"] = None
```

Comment:
Let's move this logic into its own method, and use the `organizations:codecov-stacktrace-integration` flag.